aten/src/ATen/native/Math.h

```diff
@@ -1353,18 +1353,18 @@ static inline typename std::enable_if<std::is_floating_point<T>::value, T>::type
 calc_i0(T _x) {
   T x = std::abs(_x);
 
-  if (x <= 8.0) {
+  if (x <= static_cast<T>(8.0)) {
```

Comment:
How about using T{8.0} here and at other places?
This looks less noisy and it doesn't create temporary like static_cast does.

Reply:
Thanks for the review! I've updated the PR accordingly. I figured that the tests previously failed probably because I inadvertently changed one of the formulae.